app/types/filtered_column_type.rb

```diff
@@ -11,8 +11,8 @@ def cast(value)
     in NilClass # default value
       nil
     # from form
-    in { id: String, filter: String } => h
-      FilteredColumn.new(column: Column.find(h[:id]), filter: h[:filter])
+    in { id: String|Hash, filter: String } => h
```

Comment:
j'etais perdu a la 1ere passe, mais sur le sorted_column_type tout s'est expliqué. donc une tite maj du commentaire au dessus serait tip top
`# from form (id is a string) or from db (id is a hash)`